src/objects/api/filters.py

```diff
@@ -1,15 +1,67 @@
+from django.utils.translation import ugettext_lazy as _
+
 from django_filters import filters
 from vng_api_common.filtersets import FilterSet
 from vng_api_common.utils import get_help_text
 
 from objects.core.models import Object
 
+from .constants import Operators
+from .utils import display_choice_values_for_help_text, is_number
+from .validators import validate_data_attrs
+
 
 class ObjectFilterSet(FilterSet):
     type = filters.CharFilter(
         field_name="object_type", help_text=get_help_text("core.Object", "object_type")
     )
+    data_attrs = filters.CharFilter(
+        method="filter_data_attrs",
+        validators=[validate_data_attrs],
+        help_text=_(
+            """Only include objects that have attributes with certain values.
+Data filtering expressions are comma-separated and are structured as follows:
```

Comment:
I think the `data_attrs` (or `record__data__*` params should only be allowed if a `type` filter is provided, otherwise the results become messy.

Reply:
Moved to #90